tests/test_FFBbasis2D.py

```diff
@@ -138,13 +130,58 @@ def testRotate(self):
         y4 = basis.evaluate(v4)
 
         # Rotate 90
-        self.assertTrue(np.allclose(y1[0], y2[0], atol=1e-4))
+        self.assertTrue(np.allclose(y1[0], y2[0], atol=1e-5))
+
+        # 2*pi Identity
+        self.assertTrue(np.allclose(x1[0], y3[0], atol=1e-5))
+
+        # Refl (flipped using flipud)
+        self.assertTrue(np.allclose(np.flipud(x1[0]), y4[0], atol=1e-5))
+
+    def testRotateComplex(self):
+        # Now low res (8x8) had problems;
+        #  better with odd (7x7), but still not good.
+        # We'll use a higher res test image.
+        # fh = np.load(os.path.join(DATA_DIR, 'ffbbasis2d_xcoeff_in_8_8.npy'))[:7,:7]
+        # Use a real data volume to generate a clean test image.
```

Comment:
vestigial comments?

Reply:
I think they are notes about our code not working as well as I thought it should...